src/components/TextInput/index.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+
+import classnames from 'classnames';
```

Comment:
let's import `classNames`, it's similar to `classname` but goes with the camel case naming of JS! 😄 

```suggestion
import classNames from 'classnames';
```